web-server/views/layout.jade

```diff
@@ -7,19 +7,16 @@ html(lang='en')
     title Curriculum - Learners Guild
     meta(name='description', content='')
     meta(name='author', content='')
-    link(rel='apple-touch-icon', sizes='152x152', href='/apple-touch-icon.png')
-    link(rel="icon", href="/favicon.ico?v=2")
-    link(rel='icon', type='image/png', sizes='32x32', href='/favicon-32x32.png')
-    link(rel='icon', type='image/png', sizes='16x16', href='/favicon-16x16.png')
+    include icons_metadata
     link(rel='mask-icon', href='/safari-pinned-tab.svg', color='#5bbad5')
     meta(name='apple-mobile-web-app-title', content='curriculum')
     meta(name='application-name', content='curriculum')
-    meta(name='theme-color', content='#ffffff')
     link(rel='stylesheet', href='/assets/bootflat/css/bootstrap.min.css')
```

Comment:
Don't we want to replace bootflat? Or is that a next step for the branding project?